<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bg_widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1261</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bg_widget{
background-color: qlineargradient(spread:pad, x1:0.117412, y1:0.207, x2:0.794137, y2:0.777, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(170, 255, 255, 255));}</string>
    </property>
    <widget class="QLabel" name="control_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="open_btn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>260</y>
       <width>300</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>300</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="door_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>500</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cá»­a</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="op_label">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>600</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
